warning C4275: non dll-interface struct

libtorrent version (or branch): master
compiler and compiler version: visual C++2015 u3

warning C4275: non dll-interface struct 'libtorrent::disk_job_fence' used as base for dll-interface struct 'libtorrent::storage_interface' (compiling source file windows\IntegrationPage.cpp)
![2016-11-22_09-10-46](https://cloud.githubusercontent.com/assets/920705/20512915/a2147e62-b093-11e6-96da-ac6d098cbc45.png)

--- a/include/libtorrent/storage.hpp
+++ b/include/libtorrent/storage.hpp
@@ -158,7 +158,7 @@ namespace libtorrent
 	// any outstanding jobs on the torrent, then the fence is lowered
 	// and it can be performed, along with the backlog of jobs that
 	// accrued while the fence was up
-	struct TORRENT_EXTRA_EXPORT disk_job_fence
+	struct TORRENT_EXPORT disk_job_fence
 	{
 		disk_job_fence();
 		~disk_job_fence()
@@ -219,7 +219,7 @@ namespace libtorrent
 	// a specific storage, are in the cache right now. It's
 	// used for quickly being able to evict all pieces for a
 	// specific torrent
-	struct TORRENT_EXTRA_EXPORT storage_piece_set
+	struct TORRENT_EXPORT storage_piece_set
 	{
 		void add_piece(cached_piece_entry* p);
 		void remove_piece(cached_piece_entry* p);

